### ISSUE TITLE
Timeshift backups the backup data

### ISSUE THREAD

**Issue by MarcoWilli** [teejee2008/timeshift#184]
Hi there!

I just read about Timeshift in an article and found gladly a tutorial how to get it installed and running on Solus Linux. But I found out that by default the timeshift backup directory gets backed up, too. I'm not sure if this is supposed to be like that??

**Comment by teejee2008** [comment]
Please provide more details / screenshots

**Comment by teejee2008** [comment]
that folder is empty right?

**Comment by MarcoWilli** [comment]
Yes it is but I guessed it is empty because I added it to exclusion list right after first backup.

**Comment by teejee2008** [comment]
You don't need to add that to exclude list. It is excluded automatically.
